WooCommerce/Classes/ViewRelated/Orders/Order Filters/Date Range Filter/Custom Range Filter/DateRangeFilterViewController.swift

```diff
@@ -0,0 +1,231 @@
+import UIKit
+
+// MARK: - DateRangeFilterViewController
+// Allow to filter the orders by a range of dates
+//
+final class DateRangeFilterViewController: UIViewController {
+
+    @IBOutlet private weak var tableView: UITableView!
+
+    // Completion callback
+    //
+    typealias Completion = (Date?, Date?) -> Void
```

Comment:
nit: might help clarify the parameters

```suggestion
    typealias Completion = (_ startDate: Date?, _ endDate: Date?) -> Void
```